Clinical-Genomics/cgweb
# Show order name before pool name

How to test:
0. Install branch on stage on clinical-db
1. Go to stage OP
2. Login
3. Go to Items->Pools

Expected outcome:
The order name is the leftmost column, then the pool name
Every Items-view now has all columns sortable except for dates (probably behaves strangely because they are in-consistent (date sometimes, datetime sometimes))
